api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -526,6 +527,8 @@ public AuthenticationResponse fetchSignUpFormValues(ProceedContext proceedContex
         AuthenticationResponse newUserRegistrationResponse = new AuthenticationResponse();
 
         try {
+            Assert.notNull(proceedContext.getSelectProfileEnrollHref(), "Policy not configured.");
```

Comment:
This diff is acting goofy. I already PR'd this change here: https://github.com/okta/okta-idx-java/pull/187/files